### ISSUE TITLE
Move columns in UI

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Illablu/google-refine#89]
```
Gridworks should support the ability to move column positions in the UI.

```

Original issue reported on code.google.com by `iainsproat` on 23 Jun 2010 at 4:05

**Comment by GoogleCodeExporter** [comment]
```
I started by adding commands in the column header dropdown menu. They should 
unblock certain scenarios. Supporting drag and drop will require quite a bit 
more work.
```

Original comment by `dfhu...@google.com` on 7 Aug 2010 at 11:22
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed before r1326. Will be available in 2.0
```

Original comment by `dfhu...@google.com` on 27 Sep 2010 at 10:24
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `tfmorris` on 18 Sep 2012 at 2:58
- Added labels: **Milestone-2.0**
